### PR TITLE
Push any sentry_tags set in SENTRY_TAGS value

### PR DESCRIPTION
**Pull request by djowett-ftw**
@mbaechtold could we discuss this briefly tomorrow?

### CODE REVIEW

File: ftw/django_utils/settings/logging.py [hunk 1]
```diff
@@ -88,3 +90,7 @@ def post_setup(cls):
                 environment=cls.SENTRY_ENVIRONMENT,
                 ca_certs=cls.SENTRY_CACERTS,
             )
+
+        if cls.SENTRY_TAGS:
+            for tagname, tagvalue in cls.SENTRY_TAGS:
```
Comment (mbaechtold) [inline ftw/django_utils/settings/logging.py:95]:
```suggestion
            for tagname, tagvalue in cls.SENTRY_TAGS.items():
````

**Comment by djowett-ftw** [comment]
This is now rebased and deployed on kek-web. I think it's good

**Review by mbaechtold — APPROVED** [review]
Thank you!

**Comment by djowett-ftw** [comment]
Environment and tags highlighted in a recent error

Screenshot 2021-01-28 at 09 55 44

**Pull request merged**